offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/projectcreation/CreateProjectModel.groovy

```diff
@@ -0,0 +1,95 @@
+package life.qbic.portal.offermanager.components.projectcreation
+
+import com.vaadin.data.provider.DataProvider
+import com.vaadin.data.provider.ListDataProvider
+import groovy.beans.Bindable
+import life.qbic.datamodel.dtos.projectmanagement.ProjectCode
+import life.qbic.datamodel.dtos.projectmanagement.ProjectSpace
+
+/**
+ * <class short description - 1 Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since <versiontag>
+ */
+class CreateProjectModel {
```

Comment:
Should this class be removed later or do we want to keep it?

Reply:
we need it for the view state

Reply:
Do you want feedback also regarding the code or just feedback about the design of the UI?